src/icon/interfaces.ts

```diff
@@ -32,7 +32,7 @@ export interface IconProps extends BaseComponentProps {
    */
   url?: string;
   /**
-   * Specifies alternate text for a custom icon. We recommend that you provide this for accessibility.
+   * Specifies alternate text for a custom icon (using the url attribute). We recommend that you provide this for accessibility.
```

Comment:
Should we put "url" in inverted commas like: `url` ?

Reply:
+1. Otherwise, looks good.